wallet/db.c

```diff
@@ -38,7 +38,6 @@ char *dbmigrations[] = {
        PRIMARY KEY (shachain_id, pos));",
     "CREATE TABLE channels ("
     "  id INTEGER," /* chan->id */
-    "  unique_id INTEGER,"
```

Comment:
We'll have to make these changes a new migration in future, but until we release 0.6 we don't make any promise of upgradeability so this is fine.